src/client/utils/tracking/adtraction.ts

```diff
@@ -8,15 +8,17 @@ import {
   isDanishAccidentBundle,
   isStudentOffer,
   isDanishTravelBundle,
+  getMainQuote,
 } from 'pages/OfferNew/utils'
 import { NoComboTypes, DkBundleTypes } from './tracking'
 
+// Exclude single contract types since they are only sold as part of bundles
 type TypeOfContractExcludedUnused = Exclude<
   TypeOfContract | NoComboTypes | DkBundleTypes,
   'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'
 >
 
-const adtractionProductMap: Record<TypeOfContractExcludedUnused, number> = {
+const adtractionContractMap: Record<TypeOfContractExcludedUnused, number> = {
```

Comment:
Hm, no that I think about it - why do we call this `Map`? 🤔 Isn't it more like `adtractionContractValues` or something?